subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/ToPlannedTransformStepConverter.java

```diff
@@ -16,23 +16,23 @@
 
 package org.gradle.api.internal.artifacts.transform;
 
+import org.gradle.enterprise.operations.dependencies.transforms.PlannedTransformStepIdentity;
 import org.gradle.execution.plan.Node;
 import org.gradle.execution.plan.ToPlannedNodeConverter;
-import org.gradle.execution.plan.ToPlannedTaskConverter;
 import org.gradle.internal.taskgraph.CalculateTaskGraphBuildOperationType;
 
 /**
- * A {@link ToPlannedTaskConverter} for {@link TransformationNode}s.
+ * A converter from {@link TransformationNode} to {@link CalculateTaskGraphBuildOperationType.PlannedNode}.
```

Comment:
🤔 I suppose we also should rename `TransformationNode` to `TransformStepNode`, right? Though that can happen in a separate PR.

Reply:
Yes, I would do it in a separate PR, because it is a fully internal rename. The part that was relevant for the build operations API is renaming of `getTransformationNodeId()` to `getTransformStepNodeId()`.